LimeBean.Tests/BeanTests.cs

```diff
@@ -114,6 +114,17 @@ public class BeanTests {
         }
 
         [Fact]
+        public void Columns() {
+            var bean = new Bean();
+            bean["a"] = 1;
+            bean["b"] = 2;
+            bean["c"] = null;
+
+            var expect = new[] { "a", "b", "c" };
```

Comment:
In this case it probably would be simpler to just break it into 3 assertions. I was just trying to be clever :sweat_smile: